### DISCUSSION TITLE
DdependencyTrack.logging.level

### DISCUSSION THREAD

**Discussion by hectorgonzalezdeandres** [DependencyTrack/dependency-track#973] [Q&A]
Hi thanks in advance,  I am configuring my LDAP server and it doesnt work. In logs I see all fine but I need DEBUG level for seeing if is there anything incorrect. Where can I change the apiserver pod property  -DdependencyTrack.logging.level=
By default is set to info and I need to change it to DEBUG, but I can´t find where or how. I have added in values.yaml
        - name: JAVA_OPTIONS
          value: -Dalpine.application.properties=/hector/application.properties -DdependencyTrack.logging.level=DEBUG

I am watching in my pod with a ps

 1 dtrack    1h55 java -Dalpine.application.properties=/hector/application.properties -DdependencyTrack.logging.level=DEBUG -DdependencyTrack.logging.level=INFO -jar dependency-track-apiserver.war -context
 3756 dtrack    0:00 grep java

But it doesn´t work because I have 2  -DdependencyTrack.logging.level

I have installed It on my k8s cluster with evryfs-oss 1.0.2 chart version

Any help would be appreciated.

Thanks!!

**Comment by DrakezulsMinimalism** [comment]
Did you try to use 
> For Docker deployments, simply set the LOGGING_LEVEL environment variable to one of INFO, WARN, ERROR, DEBUG, or TRACE (from https://docs.dependencytrack.org/getting-started/configuration/)

Maybe this works better for your POD.
Also, which log did you look at? server.log or dependency-track.log (i.e. the terminal output)

**Reply by hectorgonzalezdeandres** [reply]
Hi, yes I did it, and now I have two loggin_levels
 java **-Dalpine.application.properties=/hector/application.properties -DdependencyTrack.logging.level=DEBUG** -DdependencyTrack.logging.level=INFO -jar dependency-track-apiserver.war -contex

I have set debug one, but what I need is set DEBUG only, because with these two I can only see INFO level.

I think that I need to see dependency-track.log, because what I want to degug is the LDAP comunication. I have configured LDAP, and does not work, but there are no errors in my logs...

Thanks!!

**Reply by ErikThorsell** [reply]
Did you ever find a way to solve this? I'm currently debugging LDAP issues too and cannot find a way to get any debug help.
